packages/aws-cdk/test/api/hotswap/logs-monitor.test.ts

```diff
@@ -0,0 +1,107 @@
+import { blue, yellow } from 'colors/safe';
```

Comment:
This file is probably in the wrong directory now (should be in `test/api/logs/logs-monitor-test.ts`).